### ISSUE TITLE
Plugin fails with 0.7.x version of ruby-kafka

### ISSUE THREAD

**Issue by zarakay** [fluent/fluent-plugin-kafka#210]
I had the plugin fail with the following stacktrace

```
2018-08-15 01:16:02 +0000 [warn]: #0 Send exception occurred: wrong number of arguments (given 6, expected 0)
2018-08-15 01:16:02 +0000 [warn]: #0 Exception Backtrace : /var/lib/gems/2.3.0/gems/ruby-kafka-0.7.0/lib/kafka/pending_message.rb:7:in `initialize'
/var/lib/gems/2.3.0/gems/fluent-plugin-kafka-0.7.4/lib/fluent/plugin/kafka_producer_ext.rb:16:in `new'
```

After some investigation I found that since this gem pins the version of `ruby-kafka` to be `< 1.0.0` by default running `sudo gem install fluent-plugin-kafka` will install the latest version of `ruby-kafka` which is `0.7.0`

Basically the error is caused by the arguments in the initializer of `pending_message` in `0.7.0` changing to positional arguments instead of normal arguments.

You can see the difference here

https://github.com/zendesk/ruby-kafka/compare/v0.6.8...v0.7.0#diff-f2ab5669a4533e5dfe4affd50afd7560L7

The fix is to pin the version or update this gem to be compatible with the newer version of ruby-kafka.

**Comment by repeatedly** [comment]
Thanks. Just released 0.7.5 to avoid this problem.
ruby-kafka v0.7 will be supported since fluent-plugin-kafka v0.8.

**Comment by zarakay** [comment]
Thanks for the quick response. I think this issue has been addressed an I believe you are looking at migrating properly so I am going to go ahead and close this issue
